contributing.md

```diff
@@ -0,0 +1,17 @@
+# Sanoma Learning Design System contributing guide
+
+We would like to welcome all contributors.
+
+These are the basic contributing rules:
+- New branch should be created from the `main` branch
+- Branch naming: `feature/[#issueNumber]-...-...` or `fix/[#issueNumber]-...-...`, eg `feature/[#21]-button-component`
+- Please use English for naming 🙂
+- Every component should have prefix `sl-` eg. `sl-button`
+- Please add unit tests for your changes
+- Commit message with scope (eg. core, storybook, website, tokens) for instance:
+    `(core): add button component`
```

Comment:
`(core)` is perhaps too vague. `(button): New component` is better I think. Also start with a capital letter :)

Reply:
The changes tool will automatically create/update a `CHANGELOG.md` for each package. So there is no need for `(core)`, since that will automatically be added to `packages/core/CHANGELOG.md`